### DISCUSSION TITLE
Different resolution for different axes

### DISCUSSION THREAD

**Discussion by raonaqul-islam** [NanoComp/meep#2779] [General]
I have a 3D geometry, which has a atomically thin 2D material layer along z-axis. Along the other two axes, the structure is large enough to use a lower resolution. Is it possible to use different resolution for three different axes? I have found an example on variable resolution here: https://github.com/fesc3555/meep_variable_resolution, but what I really need is different resolutions.

**Comment by smartalecH** [comment]
Nope. You'd have to use "tricks" like coordinate transformations to change the resolution along an axis. Meep uses the same resolution for all axes in the timestepping kernels (to keep things simple).
